src/mem/epoch/markable_atomic.rs

```diff
@@ -48,32 +53,53 @@ fn opt_owned_into_usize<T>(ptr: Option<Owned<T>>, mark: usize) -> usize {
     raw as usize | mark
 }
 
-impl<T> MarkedAtomic<T> {
-    /// Create a new, null atomic pointer.
+impl<T> MarkableAtomic<T> {
+    /// Create a new, null markable atomic pointer with a marker value of 0.
     #[cfg(feature = "nightly")]
-    pub const fn null() -> MarkedAtomic<T> {
-        MarkedAtomic {
+    pub const fn zero() -> MarkableAtomic<T> {
```

Comment:
I replaced `null` with this, and added a `usize` argument to `null`, since without that it's not possible to initialize a value of (0 as *mut _, 20) safely.